src/lib/config_parser.c

```diff
@@ -129,6 +129,11 @@ char *singularity_config_get_value(char *key) {
     return(NULL);
 }
 
+char *singularity_config_get_value_default(char *key, const char *def) {
```

Comment:
Right now, default values are created by adding a line `key = default` into the configuration file that gets distributed. Are we sure we need to add this function? The first method also ensure that two different calls to the function will always return the same value. If we add this function, it would make sense to me that we should change every call to `config_get_value` -> `config_get_value_default`. Otherwise, simply adding the default entry into the config file should do. Agree?